sphinx/writers/html.py

```diff
@@ -24,6 +24,8 @@ class HTMLWriter(Writer):  # type: ignore[misc]
     # override embed-stylesheet default value to False.
     settings_default_overrides = {'embed_stylesheet': False}
 
+    has_equations: bool = False
```

Comment:
I'm not a fan of declaring a class variable and would rather only use an instance variable if possible.